src/components/toolbox/autoSuggest/index.js

```diff
@@ -11,6 +11,7 @@ class AutoSuggest extends React.Component {
     this.state = {
       dropdownIndex: 0,
       isLoading: false,
+      isOnFocus: true,
```

Comment:
`isBlurred` and `isFocused` are more popular terms.

Reply:
true my mistake, I just fixed